jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -749,6 +749,12 @@ private static void ensureProperties(boolean isInitialization) {
 	initializedProperties.put("native.encoding", platformEncoding); //$NON-NLS-1$
 	/*[ENDIF] JAVA_SPEC_VERSION >= 17 */
 
+	/*[IF JAVA_SPEC_VERSION >= 21]*/
+	if (null == initializedProperties.get("jdk.reflect.useDirectMethodHandle")) {
+		initializedProperties.put("jdk.reflect.useDirectMethodHandle", "false");
+	}
+	/*[ENDIF] JAVA_SPEC_VERSION >= 21 */
```

Comment:
https://github.com/eclipse-openj9/openj9/issues/20155#issuecomment-2346553050 notes that `jdk.reflect.useDirectMethodHandle` is only relevant for jdk21, so either this should say `JAVA_SPEC_VERSION == 21` or https://github.com/ibmruntimes/openj9-openjdk-jdk21/pull/187 should be reopened instead of this.

Reply:
Yes, agree. Thanks a lot for pointing this out, fixed in https://github.com/eclipse-openj9/openj9/compare/42fb9abd830c8e00c0f380f4ff0a79d77b976ac8..f30df47a917fee07df9067bbf8f9dab9b4992b79